src/pretix/base/datasync/datasync.py

```diff
@@ -252,9 +252,15 @@ def get_field_value(self, inputs, mapping_entry):
         except KeyError:
             with language(self.event.settings.locale):
                 raise SyncConfigError([_(
-                    'Field "{field_name}" is not valid for {available_inputs}. Please check your {provider_name} settings.'
-                ).format(key=key, available_inputs="/".join(inputs.keys()), provider_name=self.display_name)])
-        input = inputs[field.required_input]
+                    'Field "{field_name}" does not exist. Please check your {provider_name} settings.'
+                ).format(field_name=key, provider_name=self.display_name)])
+        try:
+            input = inputs[field.required_input]
+        except KeyError:
+            with language(self.event.settings.locale):
+                raise SyncConfigError([_(
+                    'Field "{field_name}" requires {required_input}, but got only {available_inputs}. Please check your {provider_name} settings.'
```

Comment:
I am no native speaker, but swapping „only“ and „got“ around feels more correct to me.
```suggestion
                    'Field "{field_name}" requires {required_input}, but only got {available_inputs}. Please check your {provider_name} settings.'
```